src/components/EmptyTableInfo.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styles from './EmptyTableInfo.module.css'
+
+const EmptyTableInfo = ({ action, header, icon, text }) => (
+    <div className={styles.wrapper} data-test="data-test-empty-table-info">
+        <div className={styles.icon}>{icon}</div>
+        <p className={styles.header}>{header}</p>
+        <p className={styles.text}>{text}</p>
+        <div className={styles.action}>{action}</div>
```

Comment:
Perhaps you already have other use cases in mind @cooper-joe, but from what I have seen in this particular PR, I would suggest replacing this very generic `action` prop with two individual props called `onActionClick` and `actionText`. We are always showing a small secondary button and only the click handler and button label differ.

And suppose we do want this to be generic, then I wonder if another prop name, such as `footerContent` or something might be more appropriate.

Perhaps even implementing this part as the `children` could be nice.... But not quite sure about that... Could also be a bit confusing....

Reply:
Good point, I think `action` as a prop is too limiting. This type of component is still evolving as I'm putting it to work in different apps, so I think keeping it as generic as possible now will help when it's moved into UI. On reflection, I don't think it will always contain an action.

`Footer` prop works well as we already have the `Header`, so I've made that change.